src/index.html

```diff
@@ -1769,7 +1769,27 @@ <h3 class="type-l type-l--trailer">Score Card</h3>
                 options: {
                     zoom: 10,
                     maxZoom: 12,
-                }
+                },
+                additionalData: [
+                  {
+                      id: 'primary-school-enrollment',
+                      url: 'https://raw.githubusercontent.com/devinit/ug-district-dashboard-viz/dev/public/assets/data/kayunga/primary_and_kindergaten_enrolment.csv',
```

Comment:
Can we have the option to use dataID here as well, as we should be able to pick this data from the API as well? From Thatcher's documentation, we can either take the `url` or `dataID`, but the URL takes precedence.